_posts/2020-12-19-generate-mini-site-with-metanorma-cli.adoc

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: "Generate mini site with Metanorma CLI"
+categories: about
+authors:
+  -
+    name: Abu Nashir
+    email: abunashir@gmail.com
+    social_links:
+      - https://github.com/abunashir
+
+excerpt: >-
+    Generate a collection mini site using Metanorma CLI
+---
+
+A set of metanorma documents can be compiled and published as a min-site. There
+are a couple of steps involved in this process, like compiling each of these
+documents, building the collection, and generate the publishable site.
```

Comment:
Still would rather you avoid using "collection" completely. Replace with "set of documents" here